### ISSUE TITLE
Plugin is not showing up in menu?

### ISSUE THREAD

**Issue by KlavsKlavsen** [manu12121999/GIMP-Background-Remover#3]
cloned this into ~/.config/GIMP/plug-ins/gimp-background-remover - and I do not get any errors in output.
If I put background-remover.py directly in plug-ins folder - it complains over not looking at files put directly there (as notes say) - so I assume it is reading the subdir.
I've made the background-remover.py file executable - and run bash ./setup.sh as well - which installed plugins it says.
What can I do to debug why its not showing in the menu?

**Comment by Tomaasci** [comment]
I also have the same problem.

**Comment by manu12121999** [comment]
I updated the plugin and readme to work with Gimp 3.0.0. Before, it was only working with some 2.99 versions. The setup also slightly changed (e.g. you need torch 2.5), so redownload and run the script again

**Comment by WeebNetsu** [comment]
I seem to be having the same issue, running Flatpak GIMP 3.04

**Comment by henry88g** [comment]
Looking for clarity on prerequisites -  the LDF github article (https://github.com/weijun-arc/LDF) stipulates:

[Python 3.5](https://www.python.org/)
[Pytorch 1.3](http://pytorch.org/)
[OpenCV 4.0](https://opencv.org/)
[Numpy 1.15](https://numpy.org/)
[TensorboardX](https://github.com/lanpa/tensorboardX)
[Apex](https://github.com/NVIDIA/apex)

Are these minimum or specific versions?  They are quite old.

I've not yet installed all on my system, and would like to be sure about this before I try.

Thanks in advance

**Comment by manu12121999** [comment]
It also works with newer versions. No need to install them yourself, the script should take care of it.

**Comment by henry88g** [comment]
Thanks for the quick response.  I guess my problem is that I'm not sufficiently familiar with Python  virtual environments and related software installation issues, as nothing happens when I try to [INSTALL] from within GIMP  (Flatpak 3.06 on Ubuntu 24.04 with NVIDIA GPU).  Given your reply, I could try again after learning more about setting up the venv, but I've decided that I don't 'need' the plugin, and don't really want to install 1.2 GB (or more) of Torch.  I just like to use GIMP and wanted to play with background removal.

Still,  I've learned a lot in the last day or so about the use of neural networks for background removal - and even something about a label decoupling.

Thanks again.
